recipe/meta.yaml

```diff
@@ -35,6 +35,7 @@ requirements:
     - setuptools
   run:
     - python
+    - libnuma
```

Comment:
Weird. We should [get `run_exports`]( https://github.com/conda-forge/numactl-feedstock/blob/0ec6046774ab806709b3f5a1bc77491692f4e2ec/recipe/meta.yaml#L36-L37 ) from `libnuma` in `host`

https://github.com/conda-forge/linux-perf-feedstock/blob/e4b81a225099797bf5e16c773ace551556a45b3f/recipe/meta.yaml#L27

Wonder why that wasn't working 🤔

Reply:
Looks like this will be fixed with PR: https://github.com/conda-forge/numactl-feedstock/pull/17